prepare_data/src/main/groovy/org/orbisgis/osm/OSMGISLayers.groovy

```diff
@@ -243,15 +242,17 @@ static IProcess prepareHydro() {
 static IProcess loadInitialData() {
     return processFactory.create(
             "Import the data concerning a given zone and generate the 4 associated zone tables",
-            [datasource : JdbcDataSource,
+            [dbPath : String,
```

Comment:
Don't forget to check if dbPath !=null or !dbPath.isEmpty()

Reply:
Done